src/context-selection/attribute-option-combo-selector-bar-item/categories-menu.js

```diff
@@ -10,6 +11,29 @@ export default function CategoriesMenu({
     selected,
     onChange,
 }) {
+    if (categories.length === 1) {
+        const category = categories[0]
+        const values = category.categoryOptions.map(({ id, displayName }) => ({
```

Comment:
Jan-Gerke wrote this logic, so I can't speak for him directly, but I think it's okay because every categoryCombo will have at least 1 category, and every category will have at least 1 categoryOption. So I don't think it's likely to end up in a situation where we need to be defensive (it could happen if they screw around in their database...but then maybe it's better to just have the app fail at that point rather than to add handling for that?)